src/atap_widgets/concordance.py

```diff
@@ -340,7 +341,9 @@ def display_results(page: int, **kwargs):
 
             display(ipywidgets.HTML(html))
 
-        keyword_input = ipywidgets.Text(description="Keyword(s):")
+        keyword_input = ipywidgets.Text(
```

Comment:
`continuous_update` obviously has performance impacts but I think it is nice to have in an interactive tool, can we have it as an argument to `show()` so you can turn it on or off?

Reply:
I could include a toggle, but I think for this specific input field it's fine to leave `continuous_update` False, as the user just has to press enter or deselect to start the search process. I intend to meet with a user next week to get their input about this change. Should I leave this out of this PR and include it in a future change?

Reply:
Yeah ask the user whether they'd want it or not.